content/en/agent/guide/private-link.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Connect to Datadog over AWS PrivateLink
+kind: guide
+further_reading:
+    - link: 'agent/logs'
+      tag: 'Documentation'
+      text: 'Enable log collection with the Agent.'
+    - link: '/integrations/amazon_web_services/#set-up-the-datadog-lambda-function'
+      tag: 'Documentation'
+      text: 'Collect logs from your AWS services'
+---
+
+<div class="alert alert-info">
+Datadog exposes AWS PrivateLink endpoints in us-east-1.
+</div>
+
+This guide walks you through how to configure [AWS PrivateLink][1] for use with Datadog. AWS PrivateLink simplifies the security of data shared with Datadog by eliminating the exposure of data to the public internet.
+
+## Overview
+
+The overall process consists of configuring an internal endpoint in your VPC that local Datadog Agents can send data to. Your VPC endpoint is then peered with the endpoint within Datadog VPC.
+
+{{< img src="agent/guide/private_link/vpc_diagram_schema.png" alt="VPC diagram Schema" >}}
+
+## Create your VPC endpoint
+
+1. Connect to the AWS console and create a new VPC endpoint:
+   {{< img src="agent/guide/private_link/create_vpc_endpoint.png" alt="Create VPC endpoint" style="width:60%;" >}}
+2. Select **Find service by name**.
+3. Fill the _service name_ text box with the following value depending of which private link you want to setup:
+
+    {{< img src="agent/guide/private_link/vpc_service_name.png" alt="VPC service name" style="width:70%;" >}}
```

Comment:
The vpce service name here probably doesn't need to be obfuscated, since we have "public" service names now and they are listed below.  I suspect this is an artifact from the original docs shared with the initial customers who each had different service names.

Reply:
indeed, it's not a big deal tbh.. we will update screenshots in another PR